docs/source/examples/simple_landscape.myst

```diff
@@ -45,7 +45,7 @@ def variational_circuit(gamma: float) -> Circuit:
         
     q0, q1 = LineQubit.range(2)
         
-    return Circuit([CNOT(q0, q1), rx(gamma)(q1), CNOT(q0, q1)])
+    return Circuit([X(q1), CNOT(q0, q1), rx(gamma)(q1), CNOT(q0, q1), X(q1)])
```

Comment:
This made me laugh :)

Can we make it so the gates are less-obviously doing nothing? Or, even better, that they actually are doing something? I think it's fine to completely change the ansatz here because it's a contrived example.

Reply:
I added a less trivial circuit. As a byproduct, the shape of the landscape is now less boring :-)
![image](https://user-images.githubusercontent.com/46054446/122887537-4c37ea80-d341-11eb-84ac-09b01fd7736f.png)

Reply:
Nice, thanks!